packages/design-system-tokens/src/tokens/zIndex.ts

```diff
@@ -3,13 +3,11 @@
  */
 import { to, zIndexTokens } from '../lib/types';
 
-const z = to<zIndexTokens>()({
+export const z = to<zIndexTokens>()({
   deep: -99999,
   default: 1,
   drawer: 500,
   dialog: 1000,
```

Comment:
just dawned on me these tokens existed. do we use `dialog` and `drawer` still or was the need for these removed by using the `<dialog>` element?

Reply:
It looks like _only_ `--z-drawer` and `--z-dialog` are used, but they might not even be necessary like you said. We have hard-coded z-index values in our CSS, and we're not even using these values like `--z-default` and `--z-deep`. Probably worth following up on soon.